data/os/RedHat/8.yaml

```
@@ -1,3 +1,2 @@
 ---
-profile_java::pkg_list:
-  "java-1.8.0-openjdk-headless":
+profile_java::pkg_list: []
```

Don't we still want a default package here?
```suggestion
profile_java::pkg_list:
  - "java-1.8.0-openjdk-headless"
```

---

Ok, I've gone with that suggestion.

I was avoiding setting a version for RHEL 8 ([] would mean we don't install anything) until we discussed more. And I really just need to get the lock file functionality available right now.

But I was probably overthinking because if somebody doesn't like the default they can override it.

And anyway...I'm tilted back to thinking 1.8 is the correct version, because if I run 'yum install java' on a RHEL 8 machine I get java-1.8.0-openjdk (NOT headless, but still 1.8.0). So using java-1.8.0-openjdk-headless as the default for RHEL 8 makes sense to me for now.